legal-api/tests/unit/resources/v2/test_business_filings/test_filings.py

```diff
@@ -1128,6 +1151,10 @@ def _get_expected_fee_code(free, filing_name, filing_json: dict, legal_type):
         ('BC1234567', RESTORATION_LIMITED_TO_FULL_FILING, 'restoration', Business.LegalTypes.COMP.value, None, False, []),
         ('BC1234567', RESTORATION_LIMITED_TO_FULL_FILING, 'restoration', Business.LegalTypes.BC_ULC_COMPANY.value, None, False, []),
         ('BC1234567', RESTORATION_LIMITED_TO_FULL_FILING, 'restoration', Business.LegalTypes.BC_CCC.value, None, False, []),
+        ('BC1234567', CONTINUATION_OUT_FILING, 'continuationOut', Business.LegalTypes.COMP.value,
+            Business.LegalTypes.BCOMP.value, False, []),
+        ('BC1234567', CONTINUATION_OUT_FILING, 'continuationOut', Business.LegalTypes.BCOMP.value,
+            Business.LegalTypes.COMP.value, False, []),
```

Comment:
Please include tests for `Business.LegalTypes.BC_ULC_COMPANY` and `Business.LegalTypes.BC_CCC`